lib/capybara/cuprite/browser.rb

```diff
@@ -210,6 +210,17 @@ def path(node)
         evaluate_on(node: node, expression: "_cuprite.path(this)")
       end
 
+      def obscured?(node, x = nil, y = nil)
+        value = if x && y
+                  evaluate_on(node: node, expression: "_cuprite.isObscured(this, #{x}, #{y})")
+                else
+                  evaluate_on(node: node, expression: "_cuprite.isObscured(this)")
+                end
+        return true if value == true
+
+        frame_obscured_at?(*value.values_at("x", "y"))
```

Comment:
Is this for the whole frame to be obscured? maybe since the frame can be obscured it makes sense to move it upper, so that we don't check node if it's in an obscured frame?